doc/changelog.rst

```diff
@@ -28,9 +28,12 @@ A full list of changes and detailed notes can be found below:
 
 Detailed notes
 
+- Update the Github Actions runner image from `macos-10.15`` to `macos-12``. The
+former began deprecation in May 2022 and was finally removed in May 2023 (PR285_)
 - Orchestrator and Colocated DB now accept a list of interfaces to bind to. The argument name is still `interface`
   for backward compatibility reasons. (PR281_)
 
+.. _PR285: https://github.com/CrayLabs/SmartSim/pull/285
 .. _PR281: https://github.com/CrayLabs/SmartSim/pull/282
```

Comment:
I know this is not part of your PR, but `_PR281` points to `.../282`. Could you fix it (aka could you correct my mistake)?

Reply:
(this refers to the line below the one you added)

Reply:
Will do!